pages/project/list/views/index.jsx

```diff
@@ -21,16 +21,17 @@ const ExpiryDate = ({date}) => {
   const expires = moment(date);
   const diff = expires.diff(now, 'months');
   const className = classnames('notice', {
-    urgent: diff < 3,
-    warning: diff >= 3 && diff < 6
+    urgent: diff < 3
   });
 
   return <Fragment>
     { expires.format('DD MMMM YYYY') }
     {
       diff < 12 && (
         <span className={className}>
-          <Snippet diff={diff + 1}>{ diff === 0 ? 'diff.singular' : 'diff.plural' }</Snippet>
+          <Snippet diff={diff + 1}>{ diff > 2
+            ? 'diff.standard'
+            : diff === 0 ? 'diff.singular' : 'diff.plural' }</Snippet>
```

Comment:
Can we pull the logic to determine which key we're using up to the top of the component function? 

It's a bit unwieldy to be nested inside the jsx, and makes working out what's going on with the nested ternary operators quite tricky.